src/test_application/activescenes/scenarios.h

```diff
@@ -115,6 +115,13 @@ struct VehicleTest
 };
 
 
+struct UniverseTest
+{
+    static void setup_scene(CommonTestScene &rScene, osp::PkgId pkg);
```

Comment:
setup_scene should be noexcept, shouldn't it?

Or will it throw exception on failure? if so, use `noexcept(false)` to convey that that's intentional.